src/main/java/io/bf2/kafka/authorizer/CustomAclAuthorizer.java

```diff
@@ -246,6 +247,66 @@ void addAllowedListeners(Map<String, ?> configs) {
         }
     }
 
+    /**
+     * To avoid misleading users we delete principals that are explicitly targeted by CustomAclAuthorizer rules.
```

Comment:
nit: we delete bindings, not principals.

Reply:
fixed